src/models/Offer.js

```diff
@@ -35,7 +34,7 @@ const offerSchema = mongoose.Schema({
   },
   formData: {
     type: mongoose.Schema.Types.ObjectId,
-    ref: Array.from(Object.keys(forms)), // Must be one of the forms defined in forms.js
```

Comment:
So this creates collections with the name 'neub[...]wohnraum' or similar, right? 
Do we somehow assure, that we can only PUT offers with an existing city (and according formData scheme) ?

Reply:
Yeah, you're right. The city name is used for finding the right form for population (and therefore must always match the models name in mongoose).
Right now we already assure that we can't enter forms without existing city, check [this](https://github.com/Integreat/neuburg-backend/blob/4f9de7bb69acc2e74029ffd632a74e7664ff50f3/src/services/OfferService.js#L33). The `FormModel` will always be `undefined` here, which gives an error. Btw this error is not handled yet, I'll create a task for it :)

Reply:
Do we need to explicitly define the model, or would mongoose create an empty one, if it doesn't find one?

Do we need to migrate the database? (assuming there is some data in it^^)

Reply:
Although I'm not sure what mongoose would do (probably throw an error I guess) this situation does not really occur. We are first creating and saving the form for each city. That's the point we would have to handle non existing cities. Only after that we create the actual `Offer` with a link to the form.

I don't think DB migration is necessary, but I'm not sure. Afaik there are no offers yet, so it shouldn't cause any problem :D

Reply:
Alright, we should still be careful and double check when deploying this^^...